2017-06-18 13:38:06,554 - 40 - ERROR - 数据已存在，未插入数据{'musicId': '416552612', 'likeCount': '4', 'content': '"\xe4\xbd\xa0\xe6\x98\xaf\xe5\x90\xa6\xe5\xb7\xb2\xe5\x8c\x96\xe4\xbd\x9c\xe9\xa3\x8e\xe9\x9b\xa8,\xe7\xa9\xbf\xe8\xb6\x8a\xe6\x97\xb6\xe5\x85\x89\xe6\x9d\xa5\xe5\x88\xb0",\xe5\x9d\x9a\xe4\xbf\xa1\xe6\xa4\xbf\xe6\xb9\xab\xe6\x89\x8d\xe6\x98\xaf\xe5\xae\x98\xe6\x96\xb9\xe7\x9a\x84cp~', 'user': '76206763', 'commentTime': '2016-07-28 19:23:15', 'id': '189976313'}
Traceback (most recent call last):
  File "/Users/wangjiawei/PycharmProjects/MySearchEngine/DataBase.py", line 39, in insertData
    result = self.cur.execute(sql)
  File "/Library/Python/2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/Library/Python/2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1064, 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\xe4\xbd\xa0\xe6\x98\xaf\xe5\x90\xa6\xe5\xb7\xb2\xe5\x8c\x96\xe4\xbd\x9c\xe9\xa3\x8e\xe9\x9b\xa8,\xe7\xa9\xbf\xe8\xb6\x8a\xe6\x97\xb6\xe5\x85\x89\xe6\x9d\xa5\xe5\x88\xb0",\xe5\x9d\x9a\xe4\xbf\xa1\xe6\xa4\xbf\xe6\xb9\xab\xe6\x89\x8d\xe6\x98\xaf\xe5\xae\x98\xe6\x96\xb9\xe7\x9a\x84cp~"," 7\' at line 1')
